### PR TITLE
Redo account request pages

### PR DESCRIPTION
**Pull request by ChristinaLK**
🔥 🔥 🔥 

plz review! 🙏

### CODE REVIEW

**Comment by rynge** [comment]
There are plenty of references to ap7 in there, including the filenames. We can clean that up later.

File: documentation/htc_workloads/workload_planning/roadmap.md [hunk 1]
```diff
@@ -43,11 +43,11 @@ Learn more about the services provided by the OSG that can support your HTC work
 If you have not done so, apply for an account at <https://portal.osg-htc.org/application>. A Research Computing Facilitator will contact you within one business day to arrange a meeting to discuss your computational goals and to activate your account. 
 
 Users with accounts created starting in Spring 2023 have their accounts managed through a new account management service. To learn how to login to your account, visit: 
-- [Requesting an OSPool ap7.ospool.osg-htc.org Account](../../../overview/account_setup/ap7-access.md)
+- [Join and Log In to OSG Manage Access Points](../../../overview/account_setup/ap7-access.md)
```
Comment (rynge) [inline documentation/htc_workloads/workload_planning/roadmap.md:46]: `OSG Manage Access Points` is a little bit difficult to parse. Is it a managed access point? Maybe quote it: `"OSG Manage" Access Points`

File: documentation/overview/account_setup/generate-add-sshkey.md [hunk 2]
```diff
@@ -95,81 +83,13 @@ can use a key without a passphrase, but this is not recommended.
...
-2. Type the address of your assigned login node as the hostname (see "Determine which login node to use" above).
-
-3. In the left hand menu, click the "+" next to "SSH" to expand the menu.
+## Next Steps
 
-4. Click "Auth" in the "SSH" menu.
+After generating the key, you will need to upload it to a web profile. 
 
-5. Click "Browse" and specify the private key file you saved in step 5 above.
+* If your account has been created using XX, follow the instructions here: [](../ap7-access.md)
```
Comment (rynge) [inline documentation/overview/account_setup/generate-add-sshkey.md:90]: Incomplete

File: documentation/overview/account_setup/generate-add-sshkey.md [hunk 3]
```diff
@@ -95,81 +83,13 @@ can use a key without a passphrase, but this is not recommended.
...
+* If your account has been created using YY, follow the instructions here: [](../connect-access.md)
```
Comment (rynge) [inline documentation/overview/account_setup/generate-add-sshkey.md:91]: Incomplete

File: documentation/overview/account_setup/registration-and-login.md [hunk 4]
```diff
@@ -3,89 +3,57 @@ ospool:
...
-your institution in the list, or type its name to find matches. 
+To start, fill out the interest form on this OSG Portal site: 
 
-> **No Institutional Identity?**
-> 
-> If you don't have an institutional identity or can't find your institution 
-> on the provided list, either (separately) sign up for a Globus ID and follow 
-> the link for that option on this page, or contact the OSG Connect support 
-> team for guidance at support@osg-htc.org
+[OS Pool Account Request](https://portal.osg-htc.org/application)
```
Comment (rynge) [inline documentation/overview/account_setup/registration-and-login.md:24]: So I apply for an `OS Pool Account` and then will get an `OSG Manage` account. Branding seems inconsistent.

File: documentation/overview/account_setup/ap7-access.md [hunk 5]
```diff
@@ -3,19 +3,20 @@ ospool:
...
+will be using the "OSG Manage" Access Points. Do not go through the steps of this 
+guide until advised to by a Research Computing Facilitator**
 
-The major steps to getting started with this access point are:
+To join and use the "OSG Manage" Access Points (`ap40.uw.osg-htc.org`), you will go through the following steps: 
 
-1. Applying for an OSPool `ap7.ospool.osg-htc.org` Access Point account 
+1. Apply for an OSG Manage Access Point account 
 2. Have your account approved by an OSG Team member
 3. Log in to `ap7.ospool.osg-htc.org`
```
Comment (rachellombardi) [inline documentation/overview/account_setup/ap7-access.md:16]:
```suggestion
3. Log in to `ap40.uw.osg-htc.org`
```

File: documentation/overview/account_setup/ap7-access.md [hunk 6]
```diff
@@ -3,19 +3,20 @@ ospool:
...
 
-This guide will discuss how to apply for an account and login to `ap7.ospool.osg-htc.org`.
+## Request Access to "OSG Manage" Access Points
 
-## Request Access to the OSPool `ap7.ospool.osg-htc.org` Access Point
 To request access to `ap7.ospool.osg-htc.org`, submit an application using the following steps:
```
Comment (rachellombardi) [inline documentation/overview/account_setup/ap7-access.md:20]:
```suggestion
To request access to `ap40.uw.osg-htc.org`, submit an application using the following steps:
```

File: documentation/overview/account_setup/ap7-access.md [hunk 7]
```diff
@@ -40,11 +41,10 @@ To request access to `ap7.ospool.osg-htc.org`, submit an application using the f
 
 ## Account Approval by a Research Computing Facilitator
 
-Once OSG staff receive your email verification, a Research Computing Facilitator will contact you within one business day to arrange a short consultation and introduction to OSG resources. During this meeting, our staff will provide personalized start-up guidance per your specific computational research goals and activate your account.
+If a meeting has not already been scheduled with a Research Computing Facilitator, one of the facilitation team will contact you about arranging a short consultation. 
 
 Following the meeting, the Facilitator will approve your account and add your profile to any relevant OSG ‘project’ names. Once your account is ready, the Facilitator will email you with your account details including the 'username' you will use to log in to the `ap7.ospool.osg-htc.org` access point.
```
Comment (rachellombardi) [inline documentation/overview/account_setup/ap7-access.md:46]:
```suggestion
Following the meeting, the Facilitator will approve your account and add your profile to any relevant OSG ‘project’ names. Once your account is ready, the Facilitator will email you with your account details including the 'username' you will use to log in to the `ap40.uw.osg-htc.org` access point. 
```

File: documentation/overview/account_setup/ap7-access.md [hunk 8]
```diff
@@ -40,11 +41,10 @@ To request access to `ap7.ospool.osg-htc.org`, submit an application using the f
...
+If a meeting has not already been scheduled with a Research Computing Facilitator, one of the facilitation team will contact you about arranging a short consultation. 
 
 Following the meeting, the Facilitator will approve your account and add your profile to any relevant OSG ‘project’ names. Once your account is ready, the Facilitator will email you with your account details including the 'username' you will use to log in to the `ap7.ospool.osg-htc.org` access point. 
 
-
 ## Log in
 
 Once your account has been added to the `ap7.ospool.osg-htc.org` access point, you will be able to log in using a terminal or SSH program. Logging in requires authenticating your credientials using one of two options: __web authentication__ or __SSH key pair authentication__. Additional information on this process will be provided during and/or following your discussion with a Research Computing Facilitator.
```
Comment (rachellombardi) [inline documentation/overview/account_setup/ap7-access.md:50]:
```suggestion
Once your account has been added to the `ap40.uw.osg-htc.org` access point, you will be able to log in using a terminal or SSH program. Logging in requires authenticating your credientials using one of two options: __web authentication__ or __SSH key pair authentication__. Additional information on this process will be provided during and/or following your discussion with a Research Computing Facilitator.
```

File: documentation/overview/account_setup/ap7-access.md [hunk 9]
```diff
@@ -77,7 +77,7 @@ To authenticate using this approach:
 
 It is also possible to authenticate using an SSH key pair, if you prefer. Logging in using SSH keys does not require access to an internet browser to log in into the OSG Access Point, `ap7.ospool.osg-htc.org`.
```
Comment (rachellombardi) [inline documentation/overview/account_setup/ap7-access.md:78]:
```suggestion
It is also possible to authenticate using an SSH key pair, if you prefer. Logging in using SSH keys does not require access to an internet browser to log in into the OSG Access Point, `ap40.uw.osg-htc.org`. 
```

File: documentation/overview/account_setup/connect-access.md [hunk 10]
```diff
@@ -0,0 +1,111 @@
...
+`login05.osgconnect.net`,`ap20.uc.osg-htc.org`), you will go through the following steps: 
+
+1. Apply for an OSG Connect Access Point account 
+2. Have your account approved by an OSG Team member
+3. Generate an ssh key and add it to your web profile
+4. Log in to the appropriate Access Point
+
+## Apply for an OSG Connect Access Point account 
+
+If prompted by a research computing facilitator, you can apply for OSG Connect here:
```
Comment (rachellombardi) [inline documentation/overview/account_setup/connect-access.md:22]:
```suggestion
If prompted by a Research Computing Facilitator, you can apply for OSG Connect here: 
```

File: documentation/overview/account_setup/connect-access.md [hunk 11]
```diff
@@ -0,0 +1,111 @@
...
+
+After following the steps above to upload your key and it's been about fifteen minutes, you should 
+be able to log in to OSG Connect. 
+
+### Determine which login node to use
+
+Before you can connect, you will need to know which login node your account is assigned to. You can find 
+this information on your profile from the OSG Connect website.
+
+1. Go to www.osgconnect.net and sign in with your CILogin.
```
Comment (rachellombardi) [inline documentation/overview/account_setup/connect-access.md:73]:
```suggestion
1. Go to www.osgconnect.net and sign in with your institution credentials that you used to request an account. 
```

File: documentation/overview/account_setup/generate-add-sshkey.md (hunk 2 above)
Comment (rachellombardi) [inline documentation/overview/account_setup/generate-add-sshkey.md:90]:
```suggestion
* If your account was created through "OSG Manage", follow the instructions here: [](../ap7-access.md)
```

File: documentation/overview/account_setup/generate-add-sshkey.md (hunk 3 above)
Comment (rachellombardi) [inline documentation/overview/account_setup/generate-add-sshkey.md:91]:
```suggestion
* If your account was created through "OSG Connect" follow the instructions here: [](../connect-access.md)
```

File: documentation/overview/account_setup/registration-and-login.md [hunk 12]
```diff
@@ -3,89 +3,57 @@ ospool:
...
-Once you've applied to join OSG Connect as described above, an OSG Connect support 
-team member will contact you to arrange an initial orientation meeting. This meeting 
-generally takes about 20-30 minutes and is a chance to talk about your work, how it will 
-fit on the OSG, and some practical next steps for getting started. Some of these 
-next steps are also listed below. 
+Before or during the orientation meeting, you will be prompted to register 
+for an account on a specific OSPool Access Point. The current default is the 
+"OSG Manage" Access Points, with registration via https://registry.cilogon.org. 
+You will be directed to follow instructions on [this page](../ap7-access) to register 
+for an account here.
```
Comment (rachellombardi) [inline documentation/overview/account_setup/registration-and-login.md:42]:
```suggestion
"OSG Manage" Access Points.
```

Comment (rachellombardi) [reply documentation/overview/account_setup/registration-and-login.md:42]: this registry link isn't the correct link to direct them to an enrollment flow - it's just the default comanage link.

File: documentation/overview/account_setup/registration-and-login.md (hunk 4 above)
Comment (ChristinaLK) [reply documentation/overview/account_setup/registration-and-login.md:24]: You're right. :P

File: documentation/overview/account_setup/registration-and-login.md [hunk 13]
```diff
@@ -3,89 +3,57 @@ ospool:
...
+https://registry.cilogon.org, accounts on `ap40.uw.osg-htc.org`), follow instructions 
+in this guide for logging in: [Registration and Log In to OSG Manage Access Points](../ap7-access.md)
 
-Once you've gone through the steps above, you should be able to log in to the OSG Connect 
-login node. See the second half of the SSH key guide for details: 
-[How to Log Into the OSG Connect Login Node][ssh-key]
+If your account is on the "OSG Connect" Access Points (registration via 
+https://www.osgconnect.net/, accounts on `login04.osgconnect.net`, `login05.osgconnect.net`), 
+follow instructions 
+in this guide for logging in: [Registration and Log In to OSG Connect Access Points](../connect-access.md)
```
Comment (rachellombardi) [inline documentation/overview/account_setup/registration-and-login.md:56]: I'd delete this text and write something more vague that makes it so we only have to update the connect-access.md and ap7-access.md guides down the road (also avoids duplicate info). 

"Once your account is approved following your meeting with a Facilitator, you will receive an email with your account details and instructions for logging into your account. Shortly after following these instructions, users can login and begin submitting jobs to the Open Science Pool."

File: documentation/overview/account_setup/registration-and-login.md [hunk 14]
```diff
@@ -3,89 +3,57 @@ ospool:
...
 
 ## Overview of access procedure and accounting
```
[the remark this answers is not in the document]
Comment (rachellombardi) [reply documentation/overview/account_setup/registration-and-login.md:58]: I'd move this entire section to the FAQ section or to some other guide. This is really nitty-gritty information for someone who hasn't even requested an account yet. I also don't think we need 5 paragraphs to define this concept. 
- it also only talks about OSG Connect.

File: documentation/overview/account_setup/registration-and-login.md [hunk 15]
```diff
@@ -3,89 +3,57 @@ ospool:
   path: overview/account_setup/registration-and-login.md
 ---
 
-Registration and Login for OSG Connect 
+Registration and Login for OSPool Access
```
Comment (rachellombardi) [inline documentation/overview/account_setup/registration-and-login.md:6]:
```suggestion
Getting Started: Overview of Requesting OSPool Access
```

Comment (rachellombardi) [reply documentation/overview/account_setup/registration-and-login.md:6]: `Registration and Login for OSPool Access` is too similar to `Join and Log In to "OSG Connect" Access Points` - I'd like it to be more clear that this is the TRUE getting started point/guide

**Review by rachellombardi — APPROVED** [review]
A few changes, but it looks good to merge.

**Pull request merged**